### PR TITLE
Adding Ceph Community module

### PR DESCRIPTION
**Pull request by jarlah**
I needed some way to test against a real Ceph server and decided to give back to the community by maintaining it myself. Im quite active on GitHub and uses it every day, so I think it will be ok with some code scanning and dependabot PRs and some good old love.

### CODE REVIEW

**Comment by jarlah** [comment]
mentioning this here https://github.com/CleverCloud/testcontainers-ceph/issues/3 but its last commit is 2 years ago and alot in Ceph has changed regarding to how the docker images work. Before we used ceph/daemon and now it's ceph/demo, plus its on quay and not docker hub

File: modules/ceph.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    Ceph, also known as Ceph Storage, is an open-source storage platform designed to provide 
+    highly scalable object, block, and file storage in a unified system. 
+    Developed by Inktank Storage (which was acquired by Red Hat in 2014), 
+    it is intended to help organizations manage vast amounts of data with a high level of fault tolerance and seamless scalability. 
+    Ceph allows enterprises and service providers to install a distributed storage infrastructure with both performance and resilience.
+    
+    Read more: 
+     - [Ceph Documentation](https://docs.ceph.com/en/latest/start/intro/)
+     - [Ceph Container Documentation](https://github.com/ceph/ceph-container/)
```
Comment (eddumelendez) [inline modules/ceph.md:25]:
```suggestion
description: |
  This module allows provisioning the Ceph server as a standalone container within your tests based on [Ceph Demo Container](https://github.com/ceph/ceph-container).

  Ceph, also known as Ceph Storage, is an open-source storage platform designed to provide 
  highly scalable object, block, and file storage in a unified system. 
  Developed by Inktank Storage (which was acquired by Red Hat in 2014), 
  it is intended to help organizations manage vast amounts of data with a high level of fault tolerance and seamless scalability. 
  Ceph allows enterprises and service providers to install a distributed storage infrastructure with both performance and resilience.
    
  Read more: 
  - [Ceph Documentation](https://docs.ceph.com/en/latest/start/intro/)
  - [Ceph Container Documentation](https://github.com/ceph/ceph-container/)
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @jarlah ! This is now available in https://testcontainers.com/modules/ceph/

**Comment by jarlah** [comment]
Its nice to give back to the community ;) And this isn't the hardest thing to maintain (famous last words I guess)

**Pull request merged**